common/models/service-instance.json

```diff
@@ -55,6 +55,11 @@
       "type": "string",
       "description": "The number of cpus to use when starting the service on this instance"
     },
+    "enableTracing": {
```

Comment:
I'm OK with this, but it sounds like a verb/action not a boolean state... would tracingEnabled be better?